packages/playwright-core/src/server/injected/clock.ts

```diff
@@ -99,40 +99,41 @@ export class ClockController {
 
   install(time: number) {
     this._replayLogOnce();
-    this._innerSetTime(time);
+    this._innerSetTime(asWallTime(time));
   }
 
-  setSystemTime(time: number) {
+  setSystemTime(time: number): number {
     this._replayLogOnce();
-    this._innerSetTime(time);
+    this._innerSetTime(asWallTime(time));
+    return time - this._now.time;
```

Comment:
It looks like this value if always zero?

Reply:
Yeah, I don't need this return value, it is leftovers, removed.